### PR TITLE
Fixed a regression with msvc 10/11/12

### LINKED ISSUE #159: Failure on msvc-12.0
```
#include <boost/fusion/container/vector.hpp>
#include <boost/type.hpp>

int main()
{
	boost::fusion::vector<int, float> v1, v2;
	v1 = v2;
}
```

```
c:\projects\testbed2013\testbed.cpp(7): error C2036: 'boost::type' : unknown size
```

This comes up in the Spirit tests, f.ex. here:

https://ci.appveyor.com/project/boostorg/boost/build/1.0.4754-develop/job/yy8ymq7qvy73ce3c

### PR DESCRIPTION
**Pull request by Flast**
I'm not sure why they are crashed with original one...

close #159

### CODE REVIEW

File: test/sequence/github-159.cpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+==============================================================================*/
+
+#include <boost/fusion/container/vector/vector.hpp>
+#include <boost/type.hpp>
+
+int main()
+{
+    boost::fusion::vector<int, float> v1;
+    boost::fusion::vector<int, float> v2(v1);
+    v1 = v2;
```
Comment (Kojoley) [inline test/sequence/github-159.cpp]: Why `test/sequence/vector_copy.cpp` did not catch this?

Comment (Flast) [reply test/sequence/github-159.cpp]: IMO, `vector_copy.cpp` is general fusion specific feature test for copying, but this issue caused by combination with `<boost/type.hpp>`, so it's not a fusion specific.

**Comment by Flast** [comment]
Anything else?

**Comment by djowel** [comment]
Wonderful!

**Comment by Flast** [comment]
Thank you for your confirmation! I'll ask release manager to merge this patch into master.

**Pull request merged**